cypress/integration/metaSpec.js

```
@@ -0,0 +1,51 @@
+describe('On a page with meta tag imgix paramaters', () => {
+  before(() => {
+    cy.visit('/cypress/fixtures/samplePage.html');
+  });
+
+  context('The images on the page', () => {
+    beforeEach(() => {
+      cy.fixture('config.js').as('config');
+    });
+    it('All loaded and rendered', () => {
+      cy.get('img').each(($el, idx, $list) => {
+        // epect the number of image elements on the page to equal fixture's
+        idx === 0 && expect($list.length).to.equal(3);
+        // "naturalWidth" and "naturalHeight" are set when the image loads
+        expect($el[0].naturalWidth).to.be.greaterThan(0);
+      });
+    });
+
+    it('Interacted with the imgix instance', () => {
+      cy.get('img').each(($el) => {
+        expect($el.attr('ix-initialized')).to.equal('ix-initialized');
+      });
```

Same as above, we need to have a length check here so this won't silently fail